fluid-music/src/index.js

```diff
@@ -11,13 +11,15 @@ const clip = require('./fluid/clip');
 // Other Stuff
 const tab = require('./tab');
 const groove = require('./groove')
+const UDPClient = require('./FluidUDPClient');
```

Comment:
This is a little annoying, but For TLAs I've been using camel case. I.e. Tla, not TLA or Osc, not OSC. Juce is inconsistent on this area, but I'd like to keep then all the same.